### PR TITLE
developer README points to new contrib guide

### PR DESCRIPTION
**Pull request by guineveresaenger**
What it says on the box.

### CODE REVIEW

File: contributors/devel/README.md [hunk 1]
```diff
@@ -8,11 +8,8 @@ Guide](http://kubernetes.io/docs/admin/).
 
 ## The process of developing and contributing code to the Kubernetes project
 
-* **Welcome to Kubernetes (New Developer Guide)**
-  ([welcome-to-kubernetes-new-developer-guide.md](welcome-to-kubernetes-new-developer-guide.md)):
-  An introductory guide to contributing to K8s.
-
-* **On Collaborative Development** ([collab.md](collab.md)): Info on pull requests and code reviews.
+* **Contributor Guide**
+  ([Please start here](/guide/README)) to learn about how to contribute to Kuberenetes
```
Comment (cblecker) [inline contributors/devel/README.md]: I think this should be `/contributors/guide`

Comment (guineveresaenger) [reply contributors/devel/README.md]: Gah. Fixed.

File: contributors/devel/README.md [hunk 2]
```diff
@@ -8,11 +8,8 @@ Guide](http://kubernetes.io/docs/admin/).
 
 ## The process of developing and contributing code to the Kubernetes project
 
-* **Welcome to Kubernetes (New Developer Guide)**
-  ([welcome-to-kubernetes-new-developer-guide.md](welcome-to-kubernetes-new-developer-guide.md)):
-  An introductory guide to contributing to K8s.
-
-* **On Collaborative Development** ([collab.md](collab.md)): Info on pull requests and code reviews.
+* **Contributor Guide**
+  ([Please start here](/contributors/guide/README)) to learn about how to contribute to Kuberenetes
```
Comment (cblecker) [inline contributors/devel/README.md]: Still a broken link. You need to use either `/contributors/guide/README.md` specifying the .md extension, or you need to strip the README off altogether and use `/contributors/guide` or `/contributors/guide/` and they will show the directory, with the README at the bottom

**Pull request merged**